composer.json

```diff
@@ -30,7 +30,7 @@
     },
     "extra": {
         "component": "package",
-        "frameworks": ["Laravel 7"]
+        "frameworks": ["Laravel 8"]
```

Comment:
If it works with Laravel 8 out-of-the-box and still works with Laravel 7, then let's make it `"frameworks": ["Laravel 7", "Laravel 8"]`, shall we? 🙂

Reply:
Hi @andywer, updated in the latest commit.